tests/system/robot/symbolPronunciationTests.py

```diff
@@ -535,3 +538,76 @@ def _doTest(
 		[endReached, ],
 		msg=f"End reached failure. actual vs expected. With symbolLevel {symbolLevel}"
 	)
+
+
+def test_tableHeaders():
+	"""
+	Announce the correct line when placed at the end of a link at the end of a list item in a contenteditable
```

Comment:
This is a fairly specific scenario. It would be good to have a "why" for why this behaviour is being tested, like the other tests.

Reply:
Ah yep, I forgot to go back to this. The table headers is an interesting example of "contextual" / secondary content. I.E. the table header text is still sourced from content, but it is not the primary text that is being traversed by the caret / review cursor. It raises the question of what symbol level should be used for secondary content.